tests/dynamic_import/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(__dirname, {parser: 'babylon'});
```

Comment:
Didn't know you could pass `{parser: 'babylon'}`, would be good to add to the readme or make a new CONTRIBUTING.md and add this kind of info for adding tests?